12.1-async-closure-to-access-a-child/Readme.md

```diff
@@ -0,0 +1,82 @@
+# 12.1 Async Closure
+
+A common use case for _useRef_ is to access a child imperatively.
+
+# Steps
+
+- We will take as starting point sample _00 boilerplate_. Copy the content of the
+  project to a fresh folder an execute _npm install_.
+
+```bash
+npm install
+```
+
+- Let's create an edge case, we will use _useRef_ to get the element clientWidth.
+  First, we need render a div element and create the ref with _useRef_.
```

Comment:
In this example we will display the given container width, this action will be triggered when the user clicks on a button